src/api/pluginsApi.js

```diff
@@ -0,0 +1,32 @@
+/* 
+   Copyright (c) 2012-2018 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+    SPDX-License-Identifier: EPL-2.0
+    Contributors:
+        Red Hat, Inc. - initial implementation
+*/
+
+import axios from 'axios';
```

Comment:
name should not use any camelCase
pluginsApi --> plugins-api.js

camel case is not working good across Windows/Linux/Macos usually